models/articles.models.js

```diff
@@ -32,3 +32,21 @@ exports.selectArticleById = (article_id) => {
       return res.rows[0];
     });
 };
+
+exports.selectCommentsByArticleId = (article_id) => {
+  return db
+    .query(
+      `
+    SELECT * FROM comments 
+    WHERE article_id = $1
+    ORDER BY created_at DESC
+    `,
+      [article_id]
+    )
+    .then((res) => {
+      if (!res.rows.length) {
+        return Promise.reject({ status: 404, msg: "Not Found" });
```

Comment:
Unfortunately, we can't reject like this here. An article with no comments would give an error instead of an empty array. 

We need to check that the article exists to know if we throw an error or not.